modules/masterbar.php

```diff
@@ -14,7 +14,7 @@
 require dirname( __FILE__ ) . '/masterbar/masterbar.php';
 
 // In order to be able to tell if it's an AMP request or not we have to hook into parse_query at a later priority.
-add_action( 'parse_query', 'jetpack_initialize_masterbar', 99 );
+add_action( 'wp', 'jetpack_initialize_masterbar', 99 );
```

Comment:
It seems to work better (the masterbar loads well whether AMP is active or not) when I use `admin_bar_init` instead of `wp`:

```suggestion
add_action( 'admin_bar_init', 'jetpack_initialize_masterbar', 99 );
```